index.html

```diff
@@ -574,10 +574,12 @@ <h3>Use Cases and Requirements</h3>
         </p>
 
         <p>
-This specification makes no normative statements with regard to the
-conformance of roles in the ecosystem, such as <a>issuers</a>, <a>holders</a>,
-or <a>verifiers</a>, because the conformance of ecosystem roles are highly
-application, use case, and market vertical specific.
+This specification includes both required and optional fields. An <a>issuer</a>
+MUST include all required fields. A <a>verifier</a> SHOULD <a>validate</a> all
+required fields. Optional fields MAY be ignored by <a>issuers</a> and
```

Comment:
```suggestion
MUST include all required fields. A <a>verifier</a> SHOULD check that all
required fields satisfy their normative requirements as defined in this specification.
Optional fields MAY be ignored by <a>issuers</a> and
```

Here's some proposed language. There might be a simpler way to say "their normative requirements as defined in this specification".